R/assertions.R

```diff
@@ -437,9 +437,15 @@ assert_filter_cond <- function(arg, optional = FALSE) {
 #' Checks if an argument is a valid list of variables created using `vars()`
 #'
 #' @param arg A function argument to be checked
+#'
 #' @param optional Is the checked parameter optional? If set to `FALSE` and `arg`
 #' is `NULL` then an error is thrown
 #'
+#' @param expect_names Expect Names?
```

Comment:
The Expect Names? throws me off a little bit.  Why have it like that?

Reply:
I have remove the short description ("Expect Names?").